openvdb_cmd/vdb_tool/include/Util.h

```diff
@@ -0,0 +1,406 @@
+// Copyright Contributors to the OpenVDB Project
+// SPDX-License-Identifier: MPL-2.0
+
+////////////////////////////////////////////////////////////////////////////////
+///
+/// @buthor Ken Museth
+///
+/// @file Util.h
+///
+/// @brief Utility functions
+///
+////////////////////////////////////////////////////////////////////////////////
+
+#ifndef VDB_TOOL_UTIL_HAS_BEEN_INCLUDED
+#define VDB_TOOL_UTIL_HAS_BEEN_INCLUDED
+
+#include <openvdb/openvdb.h>
+
+#include <algorithm>// for std::transform
+#include <cctype> // for std::tolower
+#include <cstring>// for std::strtok
+#include <iomanip> // for std::setfill
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+#include <sys/stat.h>
+
+namespace openvdb {
+OPENVDB_USE_VERSION_NAMESPACE
+namespace OPENVDB_VERSION_NAME {
+namespace vdb_tool {
+
+/// @brief return @c true if @b pattern is contained in @b str starting from position @b pos
```

Comment:
There are a lot of global functions here. I wonder if you would consider rewriting them as class static members to be closer to this standard: https://www.openvdb.org/documentation/doxygen/codingStyle.html#sGlobalFunctions.

Reply:
this is just a guideline (that we never strictly enforce) and I would argue it doesn't apply here since these functions have little in common other than being used by the vdb_tool. More to the point, these functions are already in the `openvdb::vdb_tool` namespace